layouts/millennium_r4_sidebar.html

```diff
@@ -21,6 +21,12 @@ <h3><a href="#" class="js-expand-btn collapsed arrow-btn" data-proofer-ignore></
           <li><a href="/millennium/r4/devices/device/">Device</a></li>
         </ul>
       </li>
+      <li class="js-topic">
```

Comment:
This needs to medications to keep the sidebar alphabetized

Reply:
https://github.com/cerner/fhir.cerner.com/commit/a9eae2c4a58134dc024c8b565f159f7b9cd6773f

Reply:
@averyallen